tests/scripts/run_testenv.sh

```diff
@@ -41,6 +41,21 @@ $SCRIPT_DIR/run_testenv.py --source-file $SOURCE_FILE $@ || return $?
 source $SOURCE_FILE
 
 # Clean up
-rm $SOURCE_FILE
-unset SOURCE_FILE
-unset SCRIPT_DIR
```

Comment:
If those variables stay in the current shell we should find less conflicting names for them

Reply:
They may stay in the current shell if the user don't invoke `cleanup` after the test are finished.
I've renamed them by adding the prefix `PARSEC_TESTENV_`